doc/getting_started/index.rst

```diff
@@ -137,7 +137,15 @@ We'll also install Zephyr's multi-purpose west tool.
 
          .. code-block:: bash
 
-            brew install cmake ninja gperf ccache dfu-util qemu dtc python3
+            brew install cmake ninja gperf ccache dfu-util qemu dtc python3 libgit2
+
+         .. note::
+            Depending on previously installed Python versions, the ``python``
```

Comment:
Should this instead point to the "beyond the GSG" section? We have a lengthy explanation about Python there, and perhaps this actually belongs in there:

https://docs.zephyrproject.org/latest/guides/beyond-GSG.html#python-and-pip

Reply:
When I and Rediar Martin were testing this on Mac Catalina, then we could not proceed without installing libgit2. Not sure why is this a new requirement only on this OS, but it was needed.

Reply:
That section should probably have a paragraph about Python 3 as well, agreed. I'll add something.
But I think we need this pointer for mac in the Getting Started Guide, since it is important for everyone to know, not only for advanced users.

Reply:
Hi All,
I would strongly recommend that someone test this on a fresh OS that this change is not required. I am quite certain that this change was needed with the NCS version I pulled in January, but it could be that dependency was a mistake and might have been removed by some other fix. I do not have a MacOS at home, else I would have tested this. This change should be here unless someone confirms/test that this is not needed.